app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ def authenticate
     if current_user
       add_token_to_connection
 
-      set_user_session if current_user[:user_id].blank?
+      set_user_session if current_user['user_id'].blank?
```

Comment:
Using a symbol is faster than a string because it gets cached

Reply:
Yeah but it doesn't work... ❓

Reply:
Fair point 😂